.storybook/routes.js

```diff
@@ -37,13 +38,16 @@ const routes = {
   '/components/navigation/breadcrumbs/': 'components-navigation-breadcrumbs-react--basic',
   '/components/navigation/pagination/': 'components-navigation-pagination-react--basic',
   '/components/popups/toast/': 'components-popups-toast-react--with-popper',
-  '/components/popups/menu/': 'preview-menu-react--basic',
+  '/components/popups/menu/': 'components-popups-menu-react--basic',
   '/components/popups/modal/': 'components-popups-modal-react--basic',
   '/components/popups/popup/': 'components-popups-popup-react--basic',
   '/components/popups/tooltip/': 'components-popups-tooltip-react--default-story',
+  '/components/text/': 'components-text-react--basic-text',
   '/getting-started/for-developers/contributing/': 'welcome-contributing--page',
   '/getting-started/for-developers/resources/api-pattern-guidelines/':
     'welcome-dev-docs-api-pattern-guidelines--page',
+  '/getting-started/for-developers/resources/collection-api/':
+    'welcome-dev-docs-collections--basic',
```

Comment:
I added this entry to the `routes.js` file so we could potentially add the Collection API docs to the Canvas Site at `/getting-started/for-developers/resources/collection-api/`.